lib/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useId, isValidElement } from 'react';
+
+export interface CheckboxProps {
+  /** Name for the component */
+  name: string;
+  /** Disabled state for the component */
+  disabled?: boolean;
+  /** Value for the component */
+  value: string;
+  /** Checked state for the component */
+  checked: boolean;
+  /** Change event for the component */
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Text for the component */
+  label: React.ReactNode;
```

Comment:
Is this for a reason to be `ReactNode`? So that we can use some "label" component later here or would just `string` be enough type for this?

Reply:
It's on purpose.